backend/app/api/v1/auth.py

```
@@ -0,0 +1,189 @@
+        )
+    new_token = create_access_token(
+        subject=token_data.subject,
+        role=token_data.role,
+        email=token_data.email,
+    )
+    return TokenResponse(access_token=new_token)
+
+
+@router.get("/users", response_model=list[dict])
+async def list_users(
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.16.3)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)